spyderlib/plugins/editor.py

```diff
@@ -801,37 +802,40 @@ def get_plugin_actions(self):
         self.todo_menu.aboutToShow.connect(self.update_todo_menu)
         
         self.warning_list_action = create_action(self,
-                _("Show warning/error list"), icon='wng_list.png',
+                _("Show warning/error list"), icon=
+                ima.icon('wng_list'),

```

Comment:
This line should be

    icon=ima.icon('wng_list'),

to be easier to read :-)